src/components/Collections/Table/Table.vue

```diff
@@ -292,6 +293,20 @@ export default {
   },
   methods: {
     ...mapActions("sync", ["startLoading"]),
+    searchIfEnter(event) {
+      /**
+       * detect if enter has been pressed after entering
+       * a text to search
+       */
+      // check if the key pressed is the enter key
+      if (event.key === "Enter" || event.keyCode === 13) {
+        /**
+         * event.key is the modern way of detecting keys
+         * event.keyCode is deprecated (left here for for legacy browsers support)
+         */
+        if (this.searchString != "") this.search();
+      }
+    },
```

Comment:
Are we sure that in legacy browsers, `event.key` will not throw an error like "property key of undefined not found"?

Reply:
haan woh `event.key` sab mein hi hota hain from what I know!